configure.ac

```diff
@@ -2905,6 +2905,43 @@ template and manually set the correct values.])
     fi
 done
 
+# Internal fixed-size types
+for i in 1 2 4 8 ; do
```

Comment:
No 16-byte integers? With GCC, you could add support to `get_c_int_type()` via https://gcc.gnu.org/onlinedocs/gcc/_005f_005fint128.html